src/JuliaGPsDocs.jl

```diff
@@ -130,8 +133,9 @@ Start background processes to render the examples using the $(LITERATE) script.
 - `examples_basedir`: relative path to the root examples folder
 - `PKG_DIR`: path to the package to be developed
 - `WEBSITE`: path to the website url
+- `parallel`: run examples in parallel
 """
-function run_examples(examples, EXAMPLES_OUT, examples_basedir, PKG_DIR, WEBSITE)
+function run_examples(examples, EXAMPLES_OUT, examples_basedir, PKG_DIR, WEBSITE, parallel)
```

Comment:
this is not actually backwards compatible... why not
```suggestion
function run_examples(examples, EXAMPLES_OUT, examples_basedir, PKG_DIR, WEBSITE, parallel=true)
```
or (I would prefer this, you'd just have to also change line 90 above) as a keyword arg (there's already too many positional ones):
```suggestion
function run_examples(examples, EXAMPLES_OUT, examples_basedir, PKG_DIR, WEBSITE; parallel=true)
```

Reply:
Well, external use would probably be through generate_examples, so feel free to take or leave this as you prefer.

Reply:
Yeah that's a good point, but I thought that's the advantage of not having registered it!

Reply:
true! hahah. we would probably want to rename it to _run_examples and/or clean up the signature. anyways, merge it as you see fit : )